sim/pokemon.ts

```diff
@@ -863,13 +863,15 @@ export class Pokemon {
 		return false;
 	}
 
-	ignoringItem() {
-		return !this.getItem().isPrimalOrb && !!(
-			this.itemState.knockedOff || // Gen 3-4
-			(this.battle.gen >= 5 && !this.isActive) ||
-			(!this.getItem().ignoreKlutz && this.hasAbility('klutz')) ||
-			this.volatiles['embargo'] || this.battle.field.pseudoWeather['magicroom']
-		);
+	ignoringItem(forceKlutz = false) {
+		const ignoreKlutz = this.getItem().ignoreKlutz && !forceKlutz;
```

Comment:
Any reason not to have this the other way around?
```suggestion
		const ignoreKlutz = !forceKlutz && this.getItem().ignoreKlutz;
```